scripts/promote_staging_to_prod.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+checks=$(curl -s  "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage") | .conclusion' | head -n 1)
+if [ "$checks" == "success" ]; then
+  echo "All checks passed in most recent staging deployment!"
+else
+  echo "Some checks failed in most recent staging deployment. Please check GHA runs."
+  exit 1
+fi
+
+if [[ $(git diff HEAD) ]]; then
+  echo "Local has uncommitted changes, please commit or stash and try again."
+  exit 1
+fi
+
+echo "fetch origin branch history"
```

Comment:
L16-L34 could also be used in the GHA. I'd like to avoid having these processes evolve separately.